l10n_co_res_partner/__openerp__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+{
+    'name': 'Colombia - Terceros',
+    'category': 'Localization',
+    'version': '9.0.0.1.0',
+    'author': 'Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'maintainer': 'dominic.krimmer@gmail.com',
+    'website': 'https://www.plastinorte.com',
+    'summary': 'Colombia Terceros: Extended Partner / '
+               'Contact Module - Odoo 9.0',
+    'images': ['images/main_screenshot.png'],
+    'depends': [
+        'account',
+        'account_accountant',
```

Comment:
@dreispt 

We tend to activate this dependency in our localizations also, in colombia and latinamerica in general localize a software without proper accouinting acces does not make too much sense.

@dkrimmer84 Remove account and base such dependencies are redundant.